comms/src/outbound_message_service/message_pool_worker.rs

```diff
@@ -52,32 +56,40 @@ pub enum WorkerError {
     MessageQueueConnectionError(ConnectionError),
 }
 
-pub struct MessagePoolWorker<P, DS> {
+/// This is an instance of a single Worker thread for the Outbound Message Pool
+pub struct MessagePoolWorker<DS> {
+    config: OutboundMessagePoolConfig,
     context: Context,
     inbound_address: InprocAddress,
     message_queue_address: InprocAddress,
-    peer_manager: Arc<PeerManager<P, DS>>,
+    message_requeue_address: InprocAddress,
+    peer_manager: Arc<PeerManager<CommsPublicKey, DS>>,
+    connection_manager: Arc<ConnectionManager>,
     #[cfg(test)]
     test_sync_sender: Option<SyncSender<String>>,
 }
 
-impl<P, DS> MessagePoolWorker<P, DS>
-where
-    P: PublicKey + Hash + Send + Sync + 'static,
-    DS: DataStore + Send + Sync + 'static,
+impl<DS> MessagePoolWorker<DS>
+where DS: DataStore + Send + Sync + 'static
```

Comment:
The `DS` generic can be removed and replaced with the `CommsDataStore` from `type.rs`

Reply:
oh that's great! Done.